api4/system.go

```diff
@@ -40,7 +40,7 @@ func (api *API) InitSystem() {
 	api.BaseRoutes.ApiRoot.Handle("/notifications/ack", api.ApiSessionRequired(pushNotificationAck)).Methods("POST")
 	api.BaseRoutes.ApiRoot.Handle("/telemetry/mobile", api.ApiSessionRequired(collectMobileTelemetry)).Methods("POST")
 
-	//api.BaseRoutes.ApiRoot.Handle("/metrics", promhttp.Handler())
+	api.BaseRoutes.ApiRoot.Handle("/alerts", api.ApiHandler(sendAlertNotification)).Methods("POST")
```

Comment:
Please make this api to be callable by system admin.

Reply:
its handled now.. only users with system_admin role will be able to invoke the API